test/unit/auth/auth.spec.ts

```diff
@@ -3215,39 +3217,19 @@ AUTH_CONFIGS.forEach((testConfig) => {
         jwt.verify(token, '', { algorithms: ['none'] });
       });
 
-      it('verifyIdToken() rejects an unsigned token when only the env var is set', async () => {
+      it('verifyIdToken() rejects an unsigned token if auth emulator is unreachable', async () => {
```

Comment:
can you add a test for `verifyIdToken()` and `verifySessionCookie()` with the emulator? I don't see any unit tests for that to confirm that `checkRevoked` is set to true.

Reply:
Added those tests!